src/pages/getting-started/intermediates-tutorial.md

```diff
@@ -109,6 +109,14 @@ uint32 numWords =  1;
 - `uint16 requestConfirmations`: How many confirmations the Chainlink node should wait before responding. The longer the node waits, the more secure the random value is. It must be greater than the `minimumRequestBlockConfirmations` limit on the coordinator contract.
 - `uint32 numWords`: How many random values to request. If you can use several random values in a single callback, you can reduce the amount of gas that you spend per random value. In this example, each transaction requests one random value.
 
+
+Don't forget to add the `VRFCoordinatorV2Interface` variable to the contract. It will be initialized in the `constructor `
+
+<!-- prettier-ignore -->
+```solidity
+VRFCoordinatorV2Interface COORDINATOR;
+```
```

Comment:
Thank you for your contribution! 

Let's add this variable to the existing code block of contract variables at the start of this section, along with an explanation in the bulleted list below that.

To give an example of placement:

```solidity
VRFCoordinatorV2Interface COORDINATOR;
uint64 s_subscriptionId;
address vrfCoordinator = 0x2Ca8E0C643bDe4C2E08ab1fA0da3401AdAD7734D;
bytes32 s_keyHash = 0x79d3d8832d904592c0bf9818b621522c988bb8b0c05cdc3b15aea1b6e8db0c15;
uint32 callbackGasLimit = 40000;
uint16 requestConfirmations = 3;
uint32 numWords =  1;
```

- `VRFCoordinatorV2Interface COORDINATOR`: (_Add explanation here_)
- `uint64 s_subscriptionId`: The subscription ID that this contract uses for funding requests. Initialized in the `constructor`.
...